Patch 1

majaklajic opened the pull request:
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### Reviewers: @webdriverio/technical-committee

christian-bromann commented:
Sorry but I am not sure why this addition is useful. All these are examples and if you don't need to require a common file, just remove it from the config. Thanks!

majaklajic commented:
Not sure if you understood this correctly, but common.js is required and
babel setup doesn't work without it.
It's not something I've added for fun.

I followed steps from your tutorial (https://webdriver.io/docs/babel.html):
1. installed babel
2. added babel.config.js
3. updated mochaOpts in wdio.config file

However, this doesn't work until you manually create *helpers/common.js*
file.
You can close the PR, but your babel setup instructions are lacking this
one last crucial step...

Best,
Maja

On Fri, Mar 29, 2019 at 10:35 AM Christian Bromann <notifications@github.com>
wrote:

> Closed #3791 <https://github.com/webdriverio/webdriverio/pull/3791>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/webdriverio/webdriverio/pull/3791#event-2238687217>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ApZdYSRu-xQdECofBmd54VFqgWR9yjfdks5vbd5ZgaJpZM4cR3C8>
> .
>

christian-bromann commented:
> Not sure if you understood this correctly, but common.js is required and babel setup doesn't work without it.

common.js is not required .. you could use it if you need to do some things before the the test starts.
